docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -3919,22 +3919,42 @@ kafka:
 
   # When enabled, the fetch request MaxBytes field is computed using the
   # compressed size of previous records. When disabled, MaxBytes is computed
-  # using uncompressed bytes. Different Kafka implementations interpret MaxBytes
-  # differently.
+  # using uncompressed bytes. Different Kafka implementations-max interpret
+  # MaxBytes differently.
   # CLI flag: -ingest-storage.kafka.use-compressed-bytes-as-fetch-max-bytes
   [use_compressed_bytes_as_fetch_max_bytes: <boolean> | default = true]
 
-  # The number of concurrent ingestion streams to the TSDB head. Every tenant
-  # has their own set of streams. 0 to disable.
-  # CLI flag: -ingest-storage.kafka.ingestion-concurrency
-  [ingestion_concurrency: <int> | default = 0]
+  # The maximum number of concurrent ingestion streams to the TSDB head. Every
+  # tenant has their own set of str-maxeams. 0 to disable.
```

Comment:
```suggestion
  # tenant has their own set of str-maxeams. Set to 0 to disable.
```

Reply:
0 to disable is [very common](https://github.com/search?q=repo%3Agrafana%2Fmimir%200%20to%20disable&type=code) among config flags - I'd like to err on consistency.